packages/handlers/neo4j/yaml-config.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Handler {
+  """
+  Handler for Neo4j
+  """
+  neo4j: Neo4jHandler
+}
+
+type Neo4jHandler @md {
+  url: String!
```

Comment:
I know it's trivial, but we should have docs for those fields as well. Just because of IDEs.